### PR TITLE
feat: Implement workspace contributor page

### PR DESCRIPTION
**Pull request by zeucapua**
## Description

Adds table and grid displaying tracked contributors for a user's workspace.

## What type of PR is this? (check all applicable)

- [X] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Mobile & Desktop Screenshots/Recordings

https://github.com/open-sauced/app/assets/20603494/0c0ecec1-ba8f-4cae-a92e-973b642c7391

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

season-9-episode-22-gif

### CODE REVIEW

**Review by bdougie — APPROVED** [review]
This works for the MVP. Clicked through to confirm it works. Look forward to getting more context and insights on this page soon.

File: pages/workspaces/[workspaceId]/contributors.tsx [hunk 1]
```diff
@@ -40,7 +42,25 @@ interface WorkspaceContributorsPageProps {
 export default function WorkspaceContributorsPage({ workspace }: WorkspaceContributorsPageProps) {
   const router = useRouter();
   const range = router.query.range ? Number(router.query.range as string) : 30;
-  const { data, error: hasError } = useGetWorkspaceContributors({ workspaceId: workspace.id, range });
+  const [page, setPage] = useState(1);
```
Comment (nickytonline) [inline pages/workspaces/[workspaceId]/contributors.tsx:45]: refactor (non-blocking): I don't think we do this in the app elsewhere yet (🤔|maybe?), but the page should be in the querystring as well.

I'll create an issue for adding the `page` to all the pages that use paging.

**Review by nickytonline — APPROVED** [review]
Thanks @zeucapua! :shipit:

**Review by jpmcb — APPROVED** [review]
Very solid!

---

Non-blocking: we probably want to also add the "Add contributors" button here when there are no contributors for a workspace yet?

Screenshot 2024-02-16 at 8 24 12 AM

Can followup on that in another PR

**Review by brandonroberts — APPROVED** [review]
🚀

**Pull request merged**